crypto/encoding/amino/amino.go

```diff
@@ -30,9 +30,9 @@ func init() {
 
 	// TODO: Have amino provide a way to go from concrete struct to route directly.
```

Comment:
This file should probably be renamed or removed or something?

Reply:
it's done in another pr. it was purposeful to name the branch `day_of_reckoning` 😄